src/components/Group.vue

```diff
@@ -7,9 +7,8 @@
       type: 'Group',
       focusElement: true
     }"
-    @focus="focus"
```

Comment:
Why are you removing the focus stuff in this PR?

Reply:
It was previously used for selecting the group you wanted to delete, however it was custom functionality and deletion wasn't possible.

It's no longer needed with the changes in this PR